### ISSUE TITLE
Camera Drift

### ISSUE THREAD

**Issue by Johnnyprim3** [volotat/SD-CN-Animation#74]
Hi, is there any fix for the camera that drifts upwards? Is there a way to 'lock' the camera view down? Thanks for your time

**Comment by volotat** [comment]
Nope, there is no way to control the camera and probably never will be. The FloweR model is used to control the motion and it is supposed to produce results similar to GEN1, if everything will work out as I planned. But for now the model architecture and training pipeline are far too poor to produce anything meaningful. I could only recommend that you wait some time till the method is  matured enough, although I can not guarantee anything for the moment.
